Amigolden/src/app/components/map/map.component.ts

```diff
@@ -34,51 +44,59 @@ export class MapComponent implements OnInit {
   zoom: number;
   address: string;
   isCreating = false;
-  locationEntitiesMap = new Array<{location: Location, data: Array<any>}>();
+
+  locationEntitiesMap = new Array<{ location: Location; data: Array<any> }>();
   private geoCoder;
+  searchForm: FormGroup;
```

Comment:
Why do we need a FormGroup here?   if its just to grab out an element look at @ViewChild

Reply:
Also, this should only be changing the styles of the App,  the internal TS probably shouldn't change,

Reply:
i've changed how the search input fields appear and used the <ion-input> thats why i introduced the searchForm for picking the value